HACKERS

```diff
@@ -134,29 +133,56 @@ object will not get missing symbols when loaded.
 It also means you have to check for and skip tests that aren't relevant to this
 version.  The recommended way to do this is like:
 
- if (ivers($]) < ivers(5.6.2)) {
+ if (ivers($]) < ivers(5.6)) {           # No quotes needed
```

Comment:
not a big deal but `ivers` should be call with a string, even if using one IV works